sites/cpgnext.com/config/navigation.js

```diff
@@ -24,14 +23,14 @@ const resources = [
 const utilities = [
   { href: 'https://www.pmmimediagroup.com/pw/packaging-world', label: 'Advertise', target: '_blank' },
   // { href: 'https://www.pmmimediagroup.com/pw/reprints-and-permissions', label: 'Reprints' },
-  { href: '/page/pw-about-us', label: 'About' },
-  { href: '/page/pw-contact-us', label: 'Contact' },
+  { href: '/page/cpgnext-about-us', label: 'About' },
+  { href: '/page/cpgnext-contact-us', label: 'Contact' },
 ];
 
 const mobileMenu = {
   primary: [...topics],
   secondary: [
-    subscribe,
```

Comment:
Is there are reason we opted to remove this outright? This looks like exactly what would have been included here had `./subscribe` been updated to include the appropriate link

Reply:
@Shinsina it is replaces with the url below 👇  line 33. They are using 2 different tracking code in their dragon forms to determine which link is clicked (the footer is different)

Reply:
@jwade1327 Can we remove the `subscribe` file then? (Assuming it goes unused elsewhere), seems silly to leave a file in there entirely unused.